index.js

```diff
@@ -1,6 +1,61 @@
 /* jshint node: true */
 'use strict';
 
+var path = require('path');
+var Funnel = require('broccoli-funnel');
+var mergeTrees = require('broccoli-merge-trees');
+var VersionChecker = require('ember-cli-version-checker');
+
 module.exports = {
-  name: 'ember-a11y'
+  name: 'ember-a11y',
+
+  init: function() {
```

Comment:
Maybe we can set `this.options.addons.blacklist.push('ember-getowner-polyfill');` if it's not required.